templates/element/header.php

```diff
@@ -1,4 +1,4 @@
-<header>
+<header class="card">
     <?php
 
```

Comment:
whitespace?

Reply:
Nope, it's a style change, not just a whitespace